puppet_consulting.html

```diff
@@ -0,0 +1,144 @@
+---
+layout: singlepage
+description: "Puppet Consulting"
+author: jeff
+---
+<section id="offerings">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 text-center">
+        <h2 class="section-heading">We know Puppet</h2>
+      </div>
+    </div>
+    <p></p>
+    <div class="row text-center">
+      <div class="col-md-3">
+        <span class="fa-stack fa-4x">
+          <i class="fa fa-circle fa-stack-2x text-primary"></i>
+          <i class="fa fa-cloud fa-stack-1x fa-inverse"></i>
+        </span>
+        <h4 class="service-heading">Puppetserver Optimization</h4>
+        <p class="text-muted">
+          Whether it's 1 or 10,000 Puppetservers, we have assisted customers in diagnosing
+          service degradation and moving past memory errors. Our data-driven approach
+          to optimization will surface the metric data you need to be proactive about
+          the health of your fleet and avoid putting out fires.
+        </p>
+      </div>
+      <div class="col-md-3">
+        <span class="fa-stack fa-4x">
+          <i class="fa fa-circle fa-stack-2x text-primary"></i>
+          <i class="fa fa-code fa-stack-1x fa-inverse"></i>
+        </span>
+        <h4 class="service-heading">Module Development</h4>
+        <p class="text-muted">
+          We specialize in Puppet code development of all sizes: net-new
+          module creation, complex multi-team solutions, custom Puppet plugins,
+          type/provider development, legacy code audits, and everything else in-between.
+        </p>
+      </div>
+      <div class="col-md-3">
+        <span class="fa-stack fa-4x">
+          <i class="fa fa-circle fa-stack-2x text-primary"></i>
+          <i class="fa fa-check fa-stack-1x fa-inverse"></i>
+        </span>
+        <h4 class="service-heading">Staff Augmentation</h4>
+        <p class="text-muted">
+          Need a team member with day-to-day Puppet experience but having a hard time finding them?
+          <a href="#contact">Contact us and let's talk!</a>
+        </p>
+      </div>
+      <div class="col-md-3">
+        <span class="fa-stack fa-4x">
+          <i class="fa fa-circle fa-stack-2x text-primary"></i>
+          <i class="fa fa-gears fa-stack-1x fa-inverse"></i>
+        </span>
+        <h4 class="service-heading">Support</h4>
+        <p class="text-muted">
+          Have a specific Puppet problem and need an opinion from some folks with experience?
+          <a href="#contact">Contact us and let's talk!</a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section id="facts">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12">
+        <h2 class="section-heading text-center">Fast Facts</h2>
```

Comment:
This entire page is nearly perfect. The only thing that stands out is the bullet-point formatting of this "Fast Facts" section.

I wonder if these bullet points would flow better as just a short paragraph of introductory text right below the "We know Puppet" header.